tools/eos-label-location.in

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python3
+#
+# eos-label-location: edit the location label for metrics instrumentation daemon
+#
+# Copyright (C) 2017 Endless Mobile, Inc.
+# Authors:
+#  Robert McQueen <rob@endlessm.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import argparse
+import configparser
+import os
+
+class LabelLocation(object):
+    CONFIG_DIR = '@sysconfdir@/metrics/'
+    CONFIG_FILE = CONFIG_DIR + 'location.conf'
```

Comment:
abandoned the directory special casing so the full path is just hardcoded as the default to -f